server/seedDB.js

```diff
@@ -0,0 +1,221 @@
+    userType: DONOR,
+    userOptions: {
+      budget: 50000,
+    },
+  },
+  {
+    username: 'supermoms',
+    name: {
+      first: 'Stella',
+      middle: 'Gotz',
+      last: 'Groove',
+    },
+    email: 'test@test.com',
+    password: 'password',
+    phone: '555-555-5555',
+    userType: FAMILY,
+    userOptions: {
+      size: 4,
+    },
+  },
+];
+
+mongoose.connect(localDB, {
+  useMongoClient: true,
+});
+
+mongoose.Promise = global.Promise;
+
+const db = mongoose.connection;
+db.on('error', console.error.bind(console, 'MongoDB connection error: '));
+
+async function clearDatabase() {
+  // Clear any existing entries in the database
+  await Donor.remove({}, (err, p) => {
+    if (err) {
+      console.error(`Error deleting Donors. Error: ${err}`);
+    }
+    console.log(`Successfully deleted ${p} Donors`);
+  });
+  await Family.remove({}, (err, p) => {
+    if (err) {
+      console.error(`Error deleting Families. Error: ${err}`);
+    }
+    console.log(`Successfully deleted ${p} Families`);
+  });
+  await Name.remove({}, (err, p) => {
+    if (err) {
+      console.error(`Error deleting Names. Error: ${err}`);
+    }
+    console.log(`Successfully deleted ${p} Names`);
+  });
+  await Organizer.remove({}, (err, p) => {
+    if (err) {
+      console.error(`Error deleting Organizers. Error: ${err}`);
+    }
+    console.log(`Successfully deleted ${p} Organizers`);
+  });
+  await User.remove({}, (err, p) => {
+    if (err) {
+      console.error(`Error deleting Users. Error: ${err}`);
+    }
+    console.log(`Successfully deleted ${p} Users`);
+  });
+  await Wishlist.remove({}, (err, p) => {
+    if (err) {
+      console.error(`Error deleting Wishlist. Error: ${err}`);
+    }
+    console.log(`Successfully deleted ${p} Wishlist`);
+  });
+}
+
+const familyCreate = async (headOfFamilyNameModel, wishlist, size) => {
+  const newWishlist = wishlist ? new Wishlist({ list: wishlist }) : undefined;
+
+  const newFamily = new Family({
+    name: headOfFamilyNameModel,
+    wishlist: newWishlist,
+    size,
+  });
+
+  await newFamily.save();
+
+  return newFamily;
+};
+
+const donorCreate = async (nameModel, budget, matchedFamily) => {
+  const newDonor = new Donor({ name: nameModel, budget, matchedFamily });
+
+  await newDonor.save();
+  return newDonor;
+};
+
+const organizerCreate = async (nameModel, organization) => {
+  const newOrganizer = new Organizer({ name: nameModel, organization });
+
+  await newOrganizer.save();
+  return newOrganizer;
+};
+
+/*
+  Make the users based on MOCK_USERS
+  Creates references to family, donor, or organizer as needed to determine user type
+*/
+const userCreate = async (
+  username,
+  name = {},
+  email,
+  password,
+  phone,
+  userType,
+  userOptions
+) => {
+  const nameModel = new Name(name);
+
+  await nameModel.save((err, name) => {
+    if (err) {
+      console.error(`Error creating Name.  Error: ${err}`);
+      return;
+    }
+    console.log('Successfully created name');
```

Comment:
If you're passing `err` and `name` into this await, should you be still using name?

Should this line be 
```
console.log(`Successfully created ${name}`);
``` 
?

Reply:
if you're using ${name} , then aren't you using name?

Reply:
I think it should be something like `console.log(`Successfully created name: ${name}`);`  Similarly with other `.save()` method calls (will update).

Reply:
Sounds good. I can agree with
```
console.log(`Successfully created name: ${name}`);
```

Would make more sense than `Successfully created Josh.`